<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNewOp</class>
 <widget class="QWidget" name="HNewOp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuova Operazione</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/icons.qrc">
    <normaloff>:/Upload.PNG</normaloff>:/Upload.PNG</iconset>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>407</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerca prodotto</string>
   </property>
  </widget>
  <widget class="QListView" name="lvprodotti">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>244</y>
     <width>1201</width>
     <height>157</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>471</width>
     <height>200</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Tipoprodotto</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbTipoLotto"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Unità di misura</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbUm"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Scadenza</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leG">
            <property name="inputMask">
             <string/>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>GG</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leM">
          <property name="inputMask">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>MM</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="leY">
          <property name="inputMask">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>YYYY</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Lotto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Quantità</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Lotto fornitore</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Note</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLineEdit" name="leLot"/>
            </item>
            <item>
             <widget class="QLineEdit" name="leQuantity">
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leLotFornitore">
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leNote"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>219</y>
     <width>118</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Carico</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Scarico</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>452</y>
     <width>1201</width>
     <height>186</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Fornitore:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>644</y>
     <width>158</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Salva</string>
      </property>
      <property name="icon">
       <iconset resource="Resources/icons.qrc">
        <normaloff>:/Printer-icon.png</normaloff>:/Printer-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Chiudi</string>
      </property>
      <property name="icon">
       <iconset resource="Resources/icons.qrc">
        <normaloff>:/Banned-Sign.PNG</normaloff>:/Banned-Sign.PNG</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>426</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leG</tabstop>
  <tabstop>leM</tabstop>
  <tabstop>leY</tabstop>
  <tabstop>leLotFornitore</tabstop>
  <tabstop>leQuantity</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>cbUm</tabstop>
  <tabstop>cbTipoLotto</tabstop>
  <tabstop>lvprodotti</tabstop>
 </tabstops>
 <resources>
  <include location="Resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
